guides/common/modules/ref_performing-a-fresh-installation-of-a-server-on-el8.adoc

```diff
@@ -0,0 +1,15 @@
+[id="Performing_a_Fresh_Installation_of_a_Server_on_el8_{context}"]
+ifdef::satellite[]
+= Performing a Fresh Installation of a Server on {RHEL} 8
```

Comment:
Is this supposed to be "Server" in the title? Maybe `{ProjectServer}`?

Reply:
Oh yeah, `ProjectServer` makes a lot of sense here!

Reply:
Actually, we should keep it generic, because later on, it speaks about both ProjectServer and Proxy Server.